guides/v2.1/cloud/project/new-relic.md

```diff
@@ -50,66 +40,77 @@ We recommend adding at least one Admin user to your New Relic account to manage
 
 {:.bs-callout .bs-callout-info}
-If you have more than three active environments using the same New Relic license key, you need to remove a license variable from an existing environment.
-Contact your Magento technical account manager or New Relic sales (sales@newrelic.com) if you want to use New Relic on more than three environments.
+If you have more than three active environments using the same New Relic license key, you need to remove a license variable from an existing environment. Contact your Magento technical account manager or New Relic sales (sales@newrelic.com) if you want to use New Relic on more than three environments.
 
-
-#### To get your New Relic license key:
+#### To find your New Relic license key:
 
 1.  Open your [account page](https://accounts.magento.cloud/user/).
 
-1.  On the Projects tab, find your project.
+1.  On the _Projects_ tab, find your project.
 
 1.  Click **View Details** to see the project provisioning information.
 
 1.  Expand the **NewRelic Service** section to view the license key.
 
 1.  Copy the license key.
 
-You can also access the license key from the Project Web interface. See [Blackfire and New Relic credentials]({{ page.baseurl }}/cloud/project/projects.html).
-
+You can also access the license key from the Magento Cloud Project interface. See [Blackfire and New Relic credentials]({{ page.baseurl }}/cloud/project/projects.html).
 
-#### To add a New Relic license key to an environment:
+### Add a license key to an environment
 
+For Starter accounts, you need to use the license key and set a variable in the environment.
 
-```bash
-magento-cloud variable:set php:newrelic.license <your-new-relic-license-key>
-```
+1.  Change to the environment that needs the license key.
 
-#### To remove a New Relic license key:
+1.  Set the variable.
 
-1.  List all variables.
+    ```bash
+    magento-cloud variable:set php:newrelic.license <newrelic-license-key>
+    ```
 
-    -  For project variables: `magento-cloud pvget`  
-    -  For environment variables: `magento-cloud vget`
+Adding a variable to an environment triggers a redeployment to update the environment variables and values. See [Working with variables]({{page.baseurl}}/cloud/env/working-with-variables.html).
 
-1.  Delete a variable.
+### Remove a license key from an environment
 
-    For project variables:
+1.  List environment variables.
 
     ```bash
-    magento-cloud project:variable:delete php:newrelic.license
+    magento-cloud vget
+    ```
+
+    Response:
+
+    ```terminal
+    +----------------------+-------------+------------------------+
+    | Name                 | Level       | Value                  |
+    +----------------------+-------------+------------------------+
+    | php:newrelic.license | environment | <newrelic-license-key> |
+    +----------------------+-------------+------------------------+
     ```
+    {: .no-copy}
+
+1.  Delete a variable.
```

Comment:
Does removing the variable also trigger redeploy?

Reply:
Why, yes it does! 🧐